.github/workflows/deploy.yml

```diff
@@ -26,9 +26,11 @@ jobs:
         run: mvn -B site --file pom.xml
       - name: Move files
         run: |
-          rm -rf docs/
```

Comment:
Are you sure that we remove this?

Reply:
Naw it's good because it'll be run from other branches, not `gh-pages` which has the docs contents